rstudio/rstudio
# support pasting of files on desktop

### Intent

Addresses #4572.

### Approach

Add a paste handler to our Ace widget instances, and use that to glean whether the user appears to be trying to paste a file into an Ace instance.

### Automated Tests

N/A

### QA Notes

On Windows desktop, try to copy a file to the clipboard. Then, in RStudio, try to paste into an editor instance. The path to the copied file should be inserted.

Note that this PR should not affect the behavior of RStudio Server / Workbench.

### Documentation

N/A

### Checklist
